test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMultipartMixedBatchReaderTests.cs

```diff
@@ -205,6 +206,42 @@ public void ReadMultipartMixedBatchRequest()
             Assert.Empty(verifyResourceStack);
         }
 
+        [Fact]
+        public void ReadMultipartMixedBatchRequestWthoutContentId()
+        {
+            var payload = Regex.Replace(batchRequestPayload, "Content-ID: .", "");

```

Comment:
we can't use the String.Replace to replace it?

Reply:
Because I want to remove all Content-ID header, not just one (in regex symbol . means any symbol)